userenvs/stream8.json

```diff
@@ -5,11 +5,11 @@
     }
   },
   "userenv": {
-    "name": "stream",
-    "label": "CentOS Stream",
+    "name": "stream8",
```

Comment:
what does happen if we stay on "stream" instead of "stream8" ? Arer you doing this to stay predictable on centos8 replacement ?

Reply:
I figured that we wanted to be more explicit in what we used and given the staleness of the non-versioned tags I figured that using them was probably unwise.